types/unified-tests.ts

```diff
@@ -91,16 +94,12 @@ processor.use([pluginWithTwoSettings])
 // $ExpectError
 processor.use(typedPlugin, settings)
 // $ExpectError
-processor.use([typedPlugin, settings])
-// $ExpectError
 processor.use(typedPlugin, typedSetting, settings)
-// $ExpectError
-processor.use([typedPlugin, typedSetting, settings])
 
 // $ExpectError
 processor.use(implicitlyTypedPlugin, settings)
 // $ExpectError
-processor.use([implicitlyTypedPlugin, settings])
+processor.use(implicitlyTypedPlugin, typedSetting, settings)
```

Comment:
Implicitly typed plugins now gain the ability to detect when the number of parameters passed is incorrect.